languages/elixir/exercises/concept/nil/.docs/hints.md

```diff
@@ -0,0 +1,23 @@
+## General
+
+- Read about `if` in the official [Getting Started Elixir guide][getting-started-if-unless] or on [elixirschool.com][elixirschool-if-unless].
+
+## 1. Print a badge for an employee
+
+- This is a base case where we can assume that none of the input data is `nil`.
+- If you need to refresh your memory about working with strings, read about them in the official [Getting Started Elixir guide][getting-started-basic-strings].
```

Comment:
I'm not sure if this hint is appropriate. The only difficulty in this step of the exercise _is_ string processing, but the concept was already covered because it's a prerequisite.

Reply:
I find hints are something that I struggle with as well, but I don't think it's a bad thing to plug the getting started guide.